### DISCUSSION TITLE
The English/Japanese example sentence for 身震い is very weird

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#13] [Errors and Corrections]
Japanese:
>「そのとおり」私は、シャワーを浴びている時に石鹸が目に入るとヒステリー状態になる男のように、娘のこの話に身震いしながら言った。

English:
> Right, I said, shivering at this recital as a man would who gets hysterical while taking a shower if a bit of soap stings his eye.

I don't think my level of Japanese (as a non-translator) is good enough to say what would be a better translation or to provide a better sentence, but as it is right now this example is really bad for a few reasons:

- The sentence is unnecessarily long. Long example sentences that rely on a lot of implied context just become unnecessarily confusing.
- The translation leaves a lot to the imagination, I assume they translated "娘の話" as "recital" which... is kinda weird
- The whole situation is borderline nonsensical, the long explanation of "like a man who got soap in his eyes during shower" doesn't sound like a good explanation of how 身震い works.

If we can find a better example sentence it would be great, but I think even just leaving it like this it does more harm than good and I'd say we should at least get rid of the sentence altogether as too low quality.

**Comment by stephenmk** [comment]
There are 8 other sentences in the corpus with 身震い. I went ahead and made this one the linked sentence for it. It's not the most natural English prose either but I guess it's good enough.

彼らは森の中で死体を見つけた時、身震いせずにはいられなかった。 
They could not help shuddering when they found a dead body in the wood.
